tests/publiccloud/storage_perf.pm

```diff
@@ -120,6 +156,33 @@ sub run {
                 values => $values
             };
             $data = influxdb_push_data($url, 'publiccloud', $data);
+            my %influx_read_args = (
+                url => $url,
+                db => 'publiccloud',
+                scenario => $href->{name},
+                os_flavor => $tags->{os_flavor},
+                os_version => $tags->{os_version}
+            );
+            if (db_has_data(%influx_read_args) {
+                foreach my $load_type (keys %$values) {
+                    $influx_read_args{load_type} = $load_type;
+                    my $current_load_type = sprintf(" load_type=%s, scenario=%s, Flavor=%s, Version=%s", $influx_read_args{load_type}, $influx_read_args{scenario}, $influx_read_args{os_flavor}, $influx_read_args{os_version});
+                    my $last_records_median = get_median_from_db(%influx_read_args, limit => $last_records);
+                    my $previous_records_median = get_median_from_db(%influx_read_args, limit => $previous_records);
+                    if (defined($previous_records_median) && defined($last_records_median) && $previous_records_median != 0 && $last_records_median != 0) {
+                        record_info('ANALYZE', "Checking " . $current_load_type);
+                        if ((abs($previous_records_median - $last_records_median) / $previous_records_median) * 100 > 10) {
```

Comment:
I guess it's ok to leave the 10% tolerance for all the pieces of data, but think we will need to re-adjust depending on which data we are analysing. 
Maybe <10% is ok for throughput, but not for latency (just to give an example without proper reasoning). I guess we will learn and improve it after getting the first results :)

Reply:
yeah I would keep this for future improvements . no matter what we will do **now** it will be random approach which would need to be improved when we will get more data